doc/quickstart.md

```diff
@@ -14,23 +14,27 @@ See [Setup](./setup.md) for other ways and more details on setting up Lean.
     del elan-init.ps1
     ```
     See the [elan repo](https://github.com/leanprover/elan) for other installation options and details.
-    
+
 1. Install [VS Code](https://code.visualstudio.com/).
 
-1. Open VS Code and install the `lean4` extension.
+1. Create a new folder called `Foo` and type:
```

Comment:
I think it is Lake convention to have a lowercase package directory & name (/cc @tydeu) (not to be confused with the .lean filename/directory *inside* the package). You might also want to use `lake new` to combine the two steps.

Reply:
Correct.

Reply:
Fixed